plugins/blockvault_client_plugin/include/eosio/blockvault_client_plugin/blockvault.hpp

```diff
@@ -0,0 +1,98 @@
+#pragma once
+#include <eosio/chain/block.hpp>
+#include <eosio/chain/block_timestamp.hpp>
+#include <stdint.h>
+#include <string_view>
+
+namespace eosio {
+namespace blockvault {
+using watermark_t = std::pair<uint32_t, eosio::chain::block_timestamp_type>;
```

Comment:
Isn't this defined elsewhere?

Reply:
producer_plugin.cpp has it, but I cannot include it as my dependency.

Reply:
It was defined here:
https://github.com/EOSIO/eos/blob/blockvault/libraries/blockvault/include/blockvault/blockvault.hpp#L8

So does this mean we're getting rid of `eos/libraries/blockvault`?